app/assets/scripts/views/location-edit.js

```diff
@@ -115,8 +117,11 @@ class LocationEdit extends React.Component {
           <li>Country: <b>{location.country}</b></li>
           {location && location.coordinates && location.coordinates.latitude && (<li>Latitude: <b>{location.coordinates.latitude}</b></li>)}
           {location && location.coordinates && location.coordinates.longitude && <li>Longitude: <b>{location.coordinates.longitude}</b></li>}
-          {metadata && metadata.siteType && <li>Location Type: <b>{metadata.siteType}</b></li>}
         </ul>
+        <div className='location-tooltip'>
+          <i className='location-tooltip-button edit-box-delete collecticons collecticons-circle-information'/>
+          <span className='location-tooltip-info'>This data is determined by the provider. Requests for changes may be made below.</span>
```

Comment:
Maybe we should say more specifically where they can request changes? Something like this?

```suggestion
          <span className='location-tooltip-info'>This data is determined by the provider. Requests for changes may be made below using the notes field and the coordinate error checkbox.</span>
```